hil/client/node.py

```diff
@@ -14,6 +15,7 @@ def list(self, is_free):
         url = self.object_url('nodes', is_free)
         return self.check_response(self.httpClient.request('GET', url))
 
+    @check_reserved_chars('node')
     def show(self, node_name):
```

Comment:
You've got a mismatch between the name you're passing to the decorator and the actual name of the parameter. Similar thing in a bunch of places below.

Reply:
I figured there were some places where the variable names wouldn't want to get out. I put 'node' there because otherwise in the error reporting it would say 'node_name' to the user.  Is that preferable?

Reply:
To settle this I'm just going to use argspec and match the variable names directly.